lms/static/sass/shared/_footer-edx.scss

```diff
@@ -83,6 +83,15 @@ footer#footer-edx-v3 {
       width: 100%;
     }
   }
+  
+  .social-media-links,
+  .mobile-app-links {
+      @extend %ui-no-list;
+      
+      li {
```

Comment:
You are still using an element as a selector, it would be preferable to use a class.